style.css

```diff
@@ -21,13 +21,13 @@ body {
   height: 100vh;
 }
 
-/* main {
+main {
   display: grid;
   grid-template-columns: column;
   width: 100%;
   height: 100vh;
   background-color: #fff;
-} */
+}
 
 .nav-container {
   display: flex;
```

Comment:
- [ ] [REQUIRED] In my opinion, you should remove the `position: fixed` property because it doesn't look good to have it sticky 👍🏼 

![image](https://user-images.githubusercontent.com/63915024/212112537-8fd3b2e2-fe41-454f-83a8-6ae944f06b64.png)

Reply:
done